core/src/trezor/lvglui/i18n/locales/zh_cn.py

```
@@ -176,7 +176,7 @@
     "存储",
     "PIN 不正确，仅剩一次尝试机会",
     "修改交易费",
-    "金额（你的花费）",
+    "金额（您的花费）",
```

Ruff (0.11.9)

String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)